feat: customize dictionary's fullindex option with metadata.toml (#1000)
xiaoyifang/goldendict-ng: website/docs/custom_dictionary.md

```

> ## Disable the fullindex of the dictionary

Reviewer: ```suggestion
## Disable full-text search for certain dictionary
```

```toml

> fullindex=false

Reviewer: ```suggestion
fts=false
```

[metadata]
name="New Name"

```

> this `fullindex` field's value can be on/off, 1/0 ,true/false etc.

Reviewer: ```suggestion
this `fts` field's value can be on/off, 1/0 ,true/false etc.
```

```

> fullindex=false

Reviewer: ```suggestion
fts=false
```

```

> will disable the current dictionary's fullindex feature.

Reviewer: ```suggestion
will disable the current dictionary's full-text search.
```

> you can check the fullindex feature on each dictionary info dialog.

Reviewer: ```suggestion
you can check the full-text search on each dictionary's info dialog.
```
